engine/launcher/src/main/scala/org/enso/launcher/internal/Environment.scala

```diff
@@ -0,0 +1,138 @@
+package org.enso.launcher.internal
+
+import java.io.File
+import java.nio.file.Path
+
+import org.enso.launcher.Logger
+
+import scala.util.Try
+
+trait Environment {
```

Comment:
I'm confused by the logic of this file. Why are you parsing `$PATH` and all that? Why not just try to run the command?

Reply:
That logic was actually added with #995. To run a plugin I just run the command, correct. But we want to treat plugins as first-class citizens and, for example, list them in the help text. To do that we need to traverse the PATH.

Reply:
mmmkay